Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -961,11 +961,11 @@ CSSCounterStyleAtRulesEnabled:
   humanReadableDescription: "Enable support for CSS @counter-style rules"
   defaultValue:
     WebKitLegacy:
-      default: false
+      default: true
     WebKit:
-      default: false
+      default: true
     WebCore:
-      default: false
+      default: true
```

Comment:
If this being set to false is what was causing the crash, we should fix the preference to properly work when disabled as well.

Reply:
I've turned on locally for testing on minibrowser and pushed by accident, but already removed.